src/Functions/array/FunctionArrayMapped.h

```diff
@@ -335,7 +335,11 @@ class FunctionArrayMapped : public IFunction
                         && column_array->getOffsets() != typeid_cast<const ColumnArray::ColumnOffsets &>(*offsets_column).getData())
                         throw Exception(
                             ErrorCodes::SIZES_OF_ARRAYS_DONT_MATCH,
-                                "Arrays passed to {} must have equal size", getName());
+                            "Arrays passed to {} must have equal size. Argument {} has size {}, but expected {}",
```

Comment:
This works incorrectly, it will always match, since this is the number of rows in the column, not number of elements in array.